### PR TITLE
Bugfix/azure-db-migrations

### LINKED ISSUE #27: CreateIdentitySchema migration is invalid.
The migration files were changed by hand, which led to a mistake - don't do that, all model changes should be done in `GameBoardDbContext.OnModelCreating`. Move the 512 MaxLength constraints there, as well as any other changes that were manually done in the `.Designer` and `ModelSnapshot` and recreate the migration. To verify that it's correct, get to @WiktorMarszalkowski to verify that he has no problems creating his migrations.

### PR DESCRIPTION
**Pull request by WiktorMarszalkowski**
Fixed migrations.

Closes #27

### CODE REVIEW

File: DataLayer/Context/GameBoardDbContext.Model.cs [hunk 1]
```diff
@@ -25,7 +25,7 @@ protected override void OnModelCreating(ModelBuilder builder)
                 entity => entity.ToTable("UserLogin"));
 
             builder.Entity<IdentityUserToken<string>>(
-                entity => entity.ToTable("UserToken"));
+                entity =>entity.ToTable("UserToken"));
```
Comment (V0ldek) [inline DataLayer/Context/GameBoardDbContext.Model.cs]: This space was innocent, bring it back.

File: DataLayer/Context/GameBoardDbContext.Model.cs [hunk 2]
```diff
@@ -12,7 +12,7 @@ protected override void OnModelCreating(ModelBuilder builder)
             builder.Entity<IdentityUser>(
                 entity => entity.ToTable("User"));
 
-            builder.Entity<IdentityRole>(
+            builder.Entity<IdentityRole>(
```
Comment (V0ldek) [inline DataLayer/Context/GameBoardDbContext.Model.cs]: ...while this space has no purpose.

File: GameBoard/Startup.cs [hunk 3]
```diff
@@ -58,7 +58,7 @@ public void ConfigureServices(IServiceCollection services)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IHostingEnvironment env)
         {
-            if (env.IsDevelopment())
+            if (env.IsDevelopment() || env.IsStaging())
```
Comment (V0ldek) [inline GameBoard/Startup.cs]: Staging should resemble production in all ways, so it shouldn't have a development exception page.

File: DataLayer/Context/GameBoardDbContext.Model.cs (hunk 1 above)
Comment (WiktorMarszalkowski) [reply DataLayer/Context/GameBoardDbContext.Model.cs]: Fixed

File: DataLayer/Context/GameBoardDbContext.Model.cs (hunk 2 above)
Comment (WiktorMarszalkowski) [reply DataLayer/Context/GameBoardDbContext.Model.cs]: Fixed

File: GameBoard/Startup.cs (hunk 3 above)
Comment (WiktorMarszalkowski) [reply GameBoard/Startup.cs]: Fixed

**Pull request merged**